starter-project-backend/src/server.ts

```diff
@@ -1,9 +1,11 @@
 import app from "./app"
 import mongoose from 'mongoose';
 
-const PORT = process.env.PORT || 8000
-const DB_URI = process.env.MONGO_URI || "";
+// const PORT = process.env.PORT || 8000
+// const DB_URI = process.env.MONGO_URI || "";
 
+const PORT = 8000
+const DB_URI = "mongodb://127.0.0.1:27017/test";
 // @ts-ignore
```

Comment:
better to use .env file to test on start_project and work on remote db

Reply:
fixed